Formula/znapzend.rb

```diff
@@ -3,6 +3,7 @@ class Znapzend < Formula
   homepage "https://www.znapzend.org/"
   url "https://github.com/oetiker/znapzend/releases/download/v0.19.1/znapzend-0.19.1.tar.gz"
   sha256 "93e3ec3c6f5cdf6973f72a6b764c49dc6545f2a0a2e0267a1382d471b930efea"
+  revision 1
```

Comment:
revision isn't needed because the bottles aren't affected

Reply:
Will `brew upgrade` still pick up the new formula if we remove `revision 1`?

Reply:
No but I don't think just adding a plist warrants an upgrade. The changes will kick in for new installs or on `brew reinstall`.

Reply:
Fair enough.